src/ios/CordovaIntercom.m

```diff
@@ -88,4 +88,12 @@ - (void) checkForUnreadMessages:(CDVInvokedUrlCommand*)command
     [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) setDeviceToken:(CDVInvokedUrlCommand*)command
+{
+    NSString* deviceToken = [command.arguments objectAtIndex:0];
+    NSData* data = [deviceToken dataUsingEncoding:NSUTF8StringEncoding];
+    [Intercom setDeviceToken:data];
```

Comment:
No need to name this variable (kinda useless), just embed it directly: `[Intercom setDeviceToken:[deviceToken dataUsingEncoding:NSUTF8StringEncoding]];`